test/integration/dialects/abstract/connection-manager.test.js

```diff
@@ -1,14 +1,12 @@
-'use strict';
-
-const chai = require('chai'),
-  expect = chai.expect,
-  deprecations = require('sequelize/lib/utils/deprecations'),
-  Support = require('../../support'),
-  sinon = require('sinon'),
-  Config = require('../../../config/config'),
-  ConnectionManager = require('sequelize/lib/dialects/abstract/connection-manager'),
-  Pool = require('sequelize-pool').Pool;
-
+const chai = require('chai');
+const { unsupportedEngine } = require('../../../../lib/utils/deprecations');
+const Support = require('../../support');
+const sinon = require('sinon');
+const ConnectionManager = require('../../../../lib/dialects/abstract/connection-manager');
```

Comment:
Why was it an absolute path before and why did you change it to a relative path? (not knowing why it was this in the first place is a valid answer)

Reply:
fixed